maintainers/scripts/haskell/mark-broken.sh

```diff
@@ -28,10 +28,11 @@ broken-packages:
   # These packages don't compile.
 EOF
 
+# clear environment here to avoid things like allowing broken builds in
```

Comment:
What does this comment mean?

Reply:
Ah, from Matrix it looks like this is to make sure that env vars like `NIXPKGS_ALLOW_BROKEN` are not set, which would mess up `maintainers/scripts/haskell/transitive-broken-packages.nix`.